docs/content/tutorial/step_13.ngdoc

```diff
@@ -21,50 +21,34 @@ In this step, we will change the way our application fetches data.
 The RESTful functionality is provided by AngularJS in the {@link ngResource ngResource} module, which
 is distributed separately from the core AngularJS framework.
 
-Since we are using [Bower][bower] to install client-side dependencies, this step updates the
-`bower.json` configuration file to include the new dependency:
+Since we are using [npm][npm] to install client-side dependencies, this step updates the
+`package.json` configuration file to include the new dependency:
 
 <br />
-**`bower.json`:**
+**`package.json`:**
 
-```
+```json
 {
   "name": "angular-phonecat",
-  "description": "A starter project for AngularJS",
-  "version": "0.0.0",
-  "homepage": "https://github.com/angular/angular-phonecat",
-  "license": "MIT",
-  "private": true,
+  ...
   "dependencies": {
-    "angular": "1.5.x",
-    "angular-mocks": "1.5.x",
```

Comment:
no mocks?

Reply:
With the switch from `bower` to `npm`, `angular-mocks` has been moved to `devDependencies (which are not shown here).